aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,179 @@
+import flowservice
+import destinationinstanceservice
+import os
+import time
+from utils import Utils
+
+class ExportDatasetToCloudStorage:
+    global flow_conn
+    global dis_conn
+    global utils
+    global username
+    def __init__(self):
+        self.username = os.getlogin()
+        self.flow_conn = flowservice.FlowService()
+        self.dis_conn = destinationinstanceservice.DestinationInstanceService()
+        self.utils = Utils()
+
+    def createDataFlowIfNotExists(self, dataset_id, compression_type, data_format, export_path, on_schedule):
+        dataflow_id = self.utils.check_if_exist("Platform", "dataflow_id")
+        if dataflow_id is not None:
+            flow = self.flow_conn.getFlow(dataflow_id)
+            source_connection_id = flow["sourceConnectionIds"][0]
+            source_connection = self.flow_conn.getSourceConnection(source_connection_id)
+            exist_dataset_id = source_connection["params"]["datasets"][0]["dataSetId"]
+            if exist_dataset_id == dataset_id and flow["state"] == "disabled":
+                self.flow_conn.postFlowAction(dataflow_id, "enable")
+            else:
+                dataflow_id = self.createDataFlow(dataset_id, compression_type, data_format, export_path, on_schedule)
+        self.createFlowRun(dataflow_id, dataset_id, on_schedule)
+
+    def createDataFlow(self, dataset_id, compression_type, data_format, export_path, on_schedule):
+        source_res = self.flow_conn.createSourceConnectionDataLake(
+            name=f"[CMLE][Week2] Featurized Dataset source connection created by {self.username}",
+            dataset_ids=[dataset_id],
+            format="parque"
+        )
+        try:
+            source_connection_id = source_res["id"]
+            print("sourceConnectionId " + source_connection_id + "has been created.")
+        except KeyError:
+            raise RuntimeError(f"Error when creating source connection: {source_res}")
+
+        connection_spec_id = "10440537-2a7b-4583-ac39-ed38d4b848e8"
+        base_connection_res = self.flow_conn.createConnection(data={
+            "name": f"[CMLE][Week2] Base Connection to DLZ created by {self.username}",
+            "auth": None,
+            "connectionSpec": {
+                "id": connection_spec_id,
+                "version": "1.0"
+            }
+        })
+        try:
+            base_connection_id = base_connection_res["id"]
+            print("baseConnectionId " + base_connection_id + "has been created")
+        except KeyError:
+            raise RuntimeError(f"Error when creating base connection: {base_connection_res}")
+
+        target_res = self.flow_conn.createTargetConnection(
+            data={
```

Comment:
should this be done in a utils module instead of this class, when I say this I mean the construction of this json